repository/src/main/java/org/apache/atlas/repository/store/graph/v2/EntityGraphRetriever.java

```diff
@@ -765,6 +765,7 @@ private AtlasEntityHeader mapVertexToAtlasEntityHeader(AtlasVertex entityVertex,
         ret.setStatus(GraphHelper.getStatus(entityVertex));
         ret.setClassificationNames(getAllTraitNames(entityVertex));
         ret.setIsIncomplete(isIncomplete);
+        ret.setHasLineage(getEntityHasLineage(entityVertex));
```

Comment:
Given that hasLineage will be set in attributes, I think we do not need this root level attribute for entity
This applies to changes made in AtlasEntity.java & AtlasEntityHeader.java

Reply:
yes, code refactoring done to remove changes in AtlasEntity.java & AtlasEntityHeader.java